packages/cwp-template-aws/template/ddb-es/apps/api/graphql/src/index.ts

```diff
@@ -112,6 +113,7 @@ export const handler = createHandler({
         }),
         createAco(),
         createAcoPageBuilderContext(),
+        createAcoFileManagerContext(),
```

Comment:
Why not just make the the ACO part of the PB and FM? Its always loaded, right? We can just push it when creating the PB or FM context.

Reply:
@brunozoric because it's not a requirement for PB or FM API to work. ACO is like an extra app on top of everything. That's why I want to keep it as separate packages, which can be replaced if necessary without digging through PB/FM/CMS respectively.